### PR TITLE
chore: imports standardisation

### PR DESCRIPTION
**Pull request by giuliamml**
## Description
Standardised imports for test files on the `/client` folder

### Dependencies

None.

## Checklist

- [x] The commit message follows our guidelines
- [x] Tests for the respective changes have been added
- [x] The code is commented, particularly in hard-to-understand areas
- [x] The labels and/or milestones were added

## Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

File: packages/client/src/addresses/__tests__/getAddress.test.ts [hunk 1]
```diff
@@ -7,8 +8,7 @@ import mswServer from '../../../tests/mswServer';
 describe('getAddress', () => {
   const expectedConfig = undefined;
   const spy = jest.spyOn(client, 'get');
-  const id = '123456';
-  const userId = 78910;
+  const id = addressId;
```
Comment (jhairabreu) [inline packages/client/src/addresses/__tests__/getAddress.test.ts:11]: same here

File: packages/client/src/addresses/__tests__/getPredictions.test.ts [hunk 2]
```diff
@@ -8,20 +9,7 @@ describe('getPredictions', () => {
...
-      type: 'Address',
-    },
-    {
-      id: 'Eh1SdWEgQXVndXN0YSwgTGlzYm9uLCBQb3J0dWdhbCIuKiwKFAoSCekeywN5NBkNEZNNGdb_qy0_EhQKEgk78-RhGjMZDRHQNpDkvesABA',
-      text: 'Rua Augusta',
-      description: 'Rua Augusta, Lisbon, Portugal',
-      type: 'Address',
-    },
-  ];
+  const response = mockPredictionsResponse;
```
Comment (jhairabreu) [inline packages/client/src/addresses/__tests__/getPredictions.test.ts:12]: same here

**Review by jhairabreu — CHANGES_REQUESTED** [review]

File: packages/client/src/checkout/__tests__/getCheckoutOrderCharge.test.ts [hunk 3]
```diff
@@ -4,12 +4,12 @@ import {
...
 } from '../types';
 import { getCheckoutOrderCharge } from '..';
+import { id } from 'tests/__fixtures__/checkout';
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/getCheckoutOrderCharge.fixtures';
 import mswServer from '../../../tests/mswServer';
 
 describe('checkout client', () => {
-  const id = 123456;
   const chargeId = 'eb92d414-68de-496e-96db-a0c6582b74d4';
```
Comment (jhairabreu) [inline packages/client/src/checkout/__tests__/getCheckoutOrderCharge.test.ts:13]: You can also move this `chargeId` to the fixtures file.

File: packages/client/src/checkout/__tests__/getDeliveryBundleUpgrades.test.ts [hunk 4]
```diff
@@ -3,14 +3,14 @@ import {
...
+import { id } from 'tests/__fixtures__/checkout';
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/getDeliveryBundleUpgrades.fixtures';
 import mswServer from '../../../tests/mswServer';
 
 describe('getDeliveryBundleUpgrades', () => {
   const spy = jest.spyOn(client, 'get');
   const expectedConfig = undefined;
-  const id = 123456;
   const deliveryBundleId = '3742-ds12-njnj-j21j';
```
Comment (jhairabreu) [inline packages/client/src/checkout/__tests__/getDeliveryBundleUpgrades.test.ts:14]: You can also move this `deliveryBundleId` to the fixtures file.

File: packages/client/src/checkout/__tests__/getUpgradeItemDeliveryProvisioning.test.ts [hunk 5]
```diff
@@ -8,10 +9,8 @@ describe('getUpgradeItemDeliveryProvisioning', () => {
   beforeEach(() => jest.clearAllMocks());
 
   const spy = jest.spyOn(client, 'get');
-  const id = 123456;
   const expectedConfig = undefined;
   const deliveryBundleId = '3742-ds12-njnj-j21j';
```
Comment (jhairabreu) [inline packages/client/src/checkout/__tests__/getUpgradeItemDeliveryProvisioning.test.ts:13]: Same for this deliveryBundleId

File: packages/client/src/checkout/__tests__/patchCheckoutCompletePayment.test.ts [hunk 6]
```diff
@@ -1,11 +1,12 @@
 import * as checkoutClient from '..';
+import { transactionId } from 'tests/__fixtures__/checkout';
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/patchCheckoutCompletePayment.fixtures';
 import mswServer from '../../../tests/mswServer';
 import type { PatchCheckoutCompletePaymentData } from '../types';
 
 describe('checkout client', () => {
-  const id = '123456';
+  const id = transactionId;
```
Comment (jhairabreu) [inline packages/client/src/checkout/__tests__/patchCheckoutCompletePayment.test.ts:9]: Use the `transactionId` directly

File: packages/client/src/checkout/__tests__/patchDeliveryBundleUpgrades.test.ts [hunk 7]
```diff
@@ -12,7 +13,6 @@ describe('patchDeliveryBundleUpgrades', () => {
 
   const spy = jest.spyOn(client, 'patch');
   const expectedConfig = undefined;
-  const id = 123456;
   const deliveryBundleId = '3742-ds12-njnj-j21j';
```
Comment (jhairabreu) [inline packages/client/src/checkout/__tests__/patchDeliveryBundleUpgrades.test.ts:16]: Same for this `deliveryBundleId`

File: packages/client/src/contents/__tests__/getContentTypes.test.ts [hunk 8]
```diff
@@ -1,4 +1,5 @@
 import { getContentTypes } from '../';
+import { types } from 'tests/__fixtures__/contents';
```
Comment (jhairabreu) [inline packages/client/src/contents/__tests__/getContentTypes.test.ts:2]: You can use `types as response` so it becomes clearer and delete L16

File: packages/client/src/returns/__tests__/getReturnPickupRescheduleRequest.test.ts [hunk 9]
```diff
@@ -1,14 +1,17 @@
...
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/getReturnPickupRescheduleRequest.fixtures';
 import join from 'proper-url-join';
 import mswServer from '../../../tests/mswServer';
 
 describe('getReturnPickupRescheduleRequest', () => {
   const spy = jest.spyOn(client, 'get');
-  const id = '123456';
-  const rescheduleRequestId = '1654321';
+  const id = mockReturnId;
```
Comment (jhairabreu) [inline packages/client/src/returns/__tests__/getReturnPickupRescheduleRequest.test.ts:14]: Use the `mockReturnId` directly and remove this line.

File: packages/client/src/returns/__tests__/getReturnPickupRescheduleRequests.test.ts [hunk 10]
```diff
@@ -1,13 +1,13 @@
...
+import { mockReturnId, responses } from 'tests/__fixtures__/returns';
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/getReturnPickupRescheduleRequests.fixtures';
 import join from 'proper-url-join';
 import mswServer from '../../../tests/mswServer';
 
 describe('getReturnPickupRescheduleRequests', () => {
   const spy = jest.spyOn(client, 'get');
-  const id = '123456';
+  const id = mockReturnId;
```
Comment (jhairabreu) [inline packages/client/src/returns/__tests__/getReturnPickupRescheduleRequests.test.ts:10]: Use the `mockReturnId` directly and remove this line.

File: packages/client/src/returns/__tests__/getReturnReferences.test.ts [hunk 11]
```diff
@@ -7,9 +8,9 @@ import mswServer from '../../../tests/mswServer';
 describe('getReturnReferences', () => {
   const name = 'ReturnNote';
   const spy = jest.spyOn(client, 'get');
-  const id = '123456';
   const expectedConfig = undefined;
   const query = { guestUserEmail: 'test@email.com' };
+  const id = mockReturnId;
```
Comment (jhairabreu) [inline packages/client/src/returns/__tests__/getReturnReferences.test.ts:13]: Use the `mockReturnId` directly and remove this line.

File: packages/client/src/returns/__tests__/getReturnsFromOrder.test.ts [hunk 12]
```diff
@@ -1,16 +1,16 @@
...
 import join from 'proper-url-join';
 import mswServer from '../../../tests/mswServer';
 
 describe('getReturnsFromOrder', () => {
   const spy = jest.spyOn(client, 'get');
-  const orderId = '123456';
   const expectedConfig = undefined;
   const query = { guestUserEmail: 'test@email.com' };
   const response = responses.get.success;
+  const orderId = mockOrderId;
```
Comment (jhairabreu) [inline packages/client/src/returns/__tests__/getReturnsFromOrder.test.ts:13]: Use the `mockOrderId` directly and remove this line.

**Review by jhairabreu — CHANGES_REQUESTED** [review]

**Review by jhairabreu — APPROVED** [review]

**Review by dnamorim — CHANGES_REQUESTED** [review]

File: packages/client/src/users/addresses/__tests__/postUserAddress.test.ts [hunk 13]
```diff
@@ -1,36 +1,11 @@
-import { AddressType } from '../types';
+import { address2, userId } from 'tests/__fixtures__/addresses';
```
[the remark this answers is not in the document]
Comment (giuliamml) [reply packages/client/src/users/addresses/__tests__/postUserAddress.test.ts:1]: yes, I think that makes sense. Will add the suggested change

[the remark this answers is not in the document]
Comment (giuliamml) [reply packages/client/src/users/addresses/__tests__/postUserAddress.test.ts:1]: yes, I think that makes sense. Will add the suggested change

File: packages/client/src/users/addresses/__tests__/putUserAddress.test.ts [hunk 14]
```diff
@@ -1,81 +1,48 @@
...
-    addressType: AddressType.Any,
-    isCurrentShipping: false,
-    isCurrentBilling: false,
-    isCurrentPreferred: false,
-    createdDate: '2021-11-04T10:34:49.891Z',
-    updatedDate: '2021-11-04T11:13:18.127Z',
-  };
-  const data = response;
+  const response = address1;
+  const data = address1;
```
[the remark this answers is not in the document]
Comment (giuliamml) [reply packages/client/src/users/addresses/__tests__/putUserAddress.test.ts:14]: agree for this too

**Review by dnamorim — APPROVED** [review]

**Review by boliveira — APPROVED** [review]

**Pull request merged**